week3/homework/src/actions/markTodo.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+{
+  function markTodo(todo, doneValue, request, response) {
+    const id = request.params.id;
+    todo
+      .readTodo(id)
+      .then(({ done }) => {
+        done = doneValue;
+        return todo.markTodo(id, done);
```

Comment:
😕 😕 😕  Why are you reading the todo only to get the done flag and then overwrite it !?